### DISCUSSION TITLE
left-over old versions of SWT natives in the repo

### DISCUSSION THREAD

**Discussion by chirontt** [eclipse-platform/eclipse.platform.swt#1088] [General]
When browsing the repo in the master branch, I'm seeing some old versions of the SWT natives (of all supported platforms) exist alongside with the latest versions. like these Linux SWT natives here:

https://github.com/eclipse-platform/eclipse.platform.swt/tree/master/binaries/org.eclipse.swt.gtk.linux.x86_64

or the Windows SWT natives here:

https://github.com/eclipse-platform/eclipse.platform.swt/tree/master/binaries/org.eclipse.swt.win32.win32.x86_64

Are they accidental left-overs when the new versions were generated recently?

**Comment by Phillipus** [comment]
I opened https://github.com/eclipse-platform/eclipse.platform.swt/issues/1095 as it might get more traction.
